src/Symfony/Component/JsonPath/JsonCrawler.php

```diff
@@ -122,12 +122,34 @@ private function evaluateName(string $name, mixed $value): array
         return \array_key_exists($name, $value) ? [$value[$name]] : [];
     }
 
+    private function splitByOperator(string $expr, string $operator): array
+    {
+        $normalizedExpr = JsonPathUtils::normalizeWhitespace($expr);
+        $pattern = '/\s*'.preg_quote($operator, '/').'\s*/';
+        $parts = preg_split($pattern, $normalizedExpr, 2);
+
+        if (2 === \count($parts)) {
+            return [trim($parts[0]), trim($parts[1])];
+        }
+
+        return [];
+    }
+
+    private function containsOperator(string $expr, string $operator): bool
```

Comment:
This method should be dropped IMO. The `str_contains` check was enough, as we don't have spaces *inside* operators (and spaces around them won't change the `contains` result)